tests/unit/flow/yaml/dummy-seg-not-random.yml

```
@@ -0,0 +1,9 @@
+!DummySegment
+requests:
+  on:
+    ControlRequest:
+      - !ControlReqDriver {}
+    [SearchRequest, TrainRequest, IndexRequest]:
+    - !SegmentDriver
+      with:
+        method: segment
```

Here is the new convention of yaml
```suggestion
jtype: DummySegment
requests:
  use_default: true
```